src/main/java/mate.academy/service/ProductService.java

```diff
@@ -1,8 +1,10 @@
 package mate.academy.service;
 
 import java.util.List;
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
```

Comment:
The `@Component` annotation should not be used on interfaces. It is intended for classes that are components in the dependency injection context.

Reply:
changes have been made